uavcan_data_types/cvra/sensor/20141.ColorTCS3472.uavcan

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
why not call it simply `20141.Color.uavcan`?

Reply:
Right, this message is quite generic. Originally, I wanted to add sensor specific configuration parameters, which became too complicated.
How about `20141.ColorRGBC.uavcan` or `20141.ColorRGBC16.uavcan`?

Reply:
`20141.ColorRGBC.uavcan` sounds good to me

Reply:
done